operate/client/src/modules/components/Operations/tests/cancel-operation.test.tsx

```diff
@@ -129,7 +129,7 @@ describe('Operations - Cancel Operation', () => {
     ).toBeInTheDocument();
 
     await user.click(
-      screen.getByRole('checkbox', {name: 'Do not show this message again'}),
+      screen.getByRole('checkbox', {name: "Don't show this message next time"}),
```

Comment:
I think we should aim to use regex for sentences for consistency and to avoid errors.

To match `src/App/Processes/ListView/InstancesTable/Toolbar/MoveAction/tests/index.test.tsx` for consistency: 

`screen.getByRole('checkbox', {name: /don't show this message next time/i}),`

Reply:
@vitorwtavares Good point. I changed all texts to regex in this file.